db/repos/blocks.js

```diff
@@ -193,13 +193,12 @@ class BlocksRepository {
 	 * @todo Add description for the params and the return value
 	 */
 	getCommonBlock(params) {
-		const query = values => {
-			values.comparePreviousBlock = values.previousBlock
-				? this.pgp.as.format('AND "previousBlock" = ${previousBlock}', values)
-				: '';
-			return sql.getCommonBlock;
-		};
-		return this.db.any(query, params);
+		return this.db.any(
```

Comment:
I think it was originally for genesis block case (because genesis block is the only one that doesn't have `previousBlock` id), but later we added a condition for genesis block here:
https://github.com/LiskHQ/lisk/blob/059935f5befd1529dbb2d69f4383dd384d167e6a/modules/blocks/process.js#L298-L304